content/en/docs/zero-code/python/operator.md

```diff
@@ -3,11 +3,25 @@ title: Using the OpenTelemetry Operator to Inject Auto-Instrumentation
 linkTitle: Operator
 aliases: [/docs/languages/python/automatic/operator]
 weight: 30
-cSpell:ignore: distro mkdir uninstrumented virtualenv
+cSpell:ignore: distro grpcio mkdir psutil uninstrumented virtualenv
 ---
 
 If you run your Python service in Kubernetes, you can take advantage of the
 [OpenTelemetry Operator](https://github.com/open-telemetry/opentelemetry-operator)
 to inject auto-instrumentation without having to modify each of your services
 directly.
 [See the OpenTelemetry Operator Auto-instrumentation docs for more details.](/docs/kubernetes/operator/automatic/)
+
+### Things to know
+
+#### Libraries with binary wheels
+
+Some Python packages we instrument (e.g. `grpcio`) or need in our
+instrumentation libraries (e.g. `psutil` for
+`opentelemetry-instrumentation-system-metrics`) may ship with some binary code.
+This binary code usually is tied to a specific C library version (glibc or musl)
+and to a specific Python version. The
+[OpenTelemetry Operator](https://github.com/open-telemetry/opentelemetry-operator)
+currently provides images for a single Python version based on the glibc C
+library so if you want to use it you'll probably need to build your own image
+operator docker image for Python auto-instrumentation.
```

Comment:
```suggestion
Some Python packages we instrument or need in our
instrumentation libraries, might ship with some binary code. This is
the case, for example, of `grpcio` and `psutil`.

The binary code is tied to a specific C library version (glibc or musl)
and to a specific Python version. The
[OpenTelemetry Operator](https://github.com/open-telemetry/opentelemetry-operator)
provides images for a single Python version based on the glibc C
library. If you want to use it you might need to build your own image
operator Docker image for Python auto-instrumentation.
```

Reply:
I would like to keep the reference to `opentelemetry-instrumentation-system-metrics` because psutil is a dependency of it and so not recognizable to the end user I think

Reply:
Something like this:

```
This is the case, for example, of `grpcio` and `psutil` (used in `opentelemetry-instrumentation-system-metrics`).
```